frontend/app/(dashboard)/updates/company/Edit.tsx

```
@@ -1,225 +1,525 @@
+        selectedRecipients.includes("administrators") || selectedRecipients.some((r) => r.includes("investor")),
+      investorTypes: selectedRecipients.filter((r) => r.startsWith("investor_")).map((r) => r.replace("investor_", "")),
+    },
+    {
+      enabled: selectedRecipients.length > 0,
+      staleTime: 2 * 60 * 1000,
+    },
+  );
+
+  const createMutation = trpc.companyUpdates.create.useMutation({
+    onError: () => {
+      alert("Failed to save update. Please try again.");
```

ESLint

[error] Unexpected alert.

(no-alert)